src/layouts/components/Header.tsx

```diff
@@ -132,18 +145,42 @@ export function Header(): JSX.Element {
       {menu && (
         <>
           <div className="fixed z-50 md:hidden">
-            <MobileMenu toggleMenu={() => setMenu(false)} />
+            <MobileMenu
+              toggleMenu={() => setMenu(false)}
+              openProposals={openProposals}
+            />
           </div>
           <div className="w-full hidden md:block md:fixed md:z-50">
-            <TabletMenu toggleMenu={() => setMenu(false)} />
+            <TabletMenu
+              toggleMenu={() => setMenu(false)}
+              openProposals={openProposals}
+            />
           </div>
         </>
       )}
     </>
   );
 }
 
-function DesktopNavbar(): JSX.Element {
+async function getOpenProposals(api: WhaleApiClient) {
+  const allProposals = await api.governance
+    .listGovProposals({
+      type: ListProposalsType.ALL,
+      status: ListProposalsStatus.VOTING,
+      all: true,
+    })
+    .catch(() => {
+      return [];
+    });
+
+  const openProposals = allProposals.filter(
+    (item) => item.status === GovernanceProposalStatus.VOTING
```

Comment:
do we still need this when we already using line 169?

Reply:
ok removed!